web/tsconfig.app.json

```diff
@@ -36,36 +29,16 @@
     "module": "esnext",
     "baseUrl": ".",
     "paths": {
-      "@/*": [
-        "./src/*"
-      ],
-      "#root/*": [
-        "./*"
-      ],
-      "app/*": [
-        "./src/app/*"
-      ],
-      "utils/*": [
-        "./src/utils/*"
-      ],
-      "api/*": [
-        "./src/api/*"
-      ],
-      "features/*": [
-        "./src/features/*"
-      ],
-      "components/*": [
-        "src/components/*"
-      ],
-      "components": [
-        "./src/components/index"
-      ],
-      "commonSlices/*": [
-        "./src/commonSlices/*"
-      ],
-    },
+      "@/*": ["./src/*"],
+      "#root/*": ["./*"],
+      "app/*": ["./src/app/*"],
+      "utils/*": ["./src/utils/*"],
+      "api/*": ["./src/api/*"],
+      "features/*": ["./src/features/*"],
+      "components/*": ["src/components/*"],
+      "components": ["./src/components/index"],
```

Comment:
prettier did this. i could undo it suppose. there's a npm format command but i don't think it's part of ci or anything. is that on purpose?

Reply:
Yeah we'll have to align that in our lint check perhaps, but we don't want any code changes during CI.